tests/func/test_datachain.py

```diff
@@ -277,7 +277,11 @@ def test_to_storage(
     file_type,
     num_threads,
 ):
-    mapper = Mock(side_effect=lambda file_path: len(file_path))
+    call_count = {"count": 0}
+
+    def mapper(file_path):
```

Comment:
what was the issue here? hash calculation for callable?

Reply:
would it be a problem with certain types of callables? e.g. custom class that dynamically imitates __call__?